kong/tools/queue_schema.lua

```diff
@@ -39,12 +39,14 @@ return Schema.define {
       initial_retry_delay = {
         type = "number",
         default = 0.01,
+        between = { 0.001, 1000000 }, -- effectively unlimited maximum
         -- description = "time in seconds before the initial retry is made for a failing batch."
         -- For each subsequent retry, the previous retry time is doubled up to `max_retry_time`
```

Comment:
unrelated to this change but should the comment say `max_retry_delay` instead of `max_retry_time`?
```suggestion
        -- For each subsequent retry, the previous retry time is doubled up to `max_retry_delay`
```

Reply:
Applied, thanks!

Reply:
Applied, thanks!